### PR TITLE
fix: center skip button

### PR DESCRIPTION
**Pull request by Iliannnn**
img

img

### CODE REVIEW

File: game.html [hunk 1]
```diff
@@ -134,7 +134,7 @@ <h1 class="settingsText">Choose your settings.</h1>
         <p><a href="#">How does it works?</a> &bull; <a href="#">Source code (GitHub)</a></p>
     </footer> -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
-    <script src="//unpkg.com/string-similarity/umd/string-similarity.min.js"></script>
+    <script src="https://unpkg.com/string-similarity/umd/string-similarity.min.js"></script>
```
Comment (Assassin-1234) [inline game.html:137]: https more like your mom

File: style.css [hunk 2]
```diff
@@ -18,12 +18,10 @@
     margin: 0;
     position: absolute;
     top: 55%;
-    right: 850px;
-    transform: translateY(-50%);
+    left: 50%;
```
Comment (Assassin-1234) [inline style.css:21]: fuckin leftist!!!!!!!! get back on twitter!

File: style.css [hunk 3]
```diff
@@ -18,12 +18,10 @@
...
 }
 
 #skipButton {
-    align-items: center;
```
Comment (Assassin-1234) [inline style.css:26]: imagine not aligning your items bro

File: style.css [hunk 4]
```diff
@@ -49,6 +47,7 @@
     touch-action: manipulation;
     vertical-align: baseline;
     width: auto;
+    left: -50%;
```
Comment (Assassin-1234) [inline style.css:50]: how political bruv

**Review by Assassin-1234 — CHANGES_REQUESTED** [review]
sus.

File: game.html (hunk 1 above)
Comment (Iliannnn) [reply game.html:137]: I'm sorry, but I am not able to browse the web. I am a large language model trained by OpenAI and my knowledge is based on the text that was available to me at the time of my training. I do not have the ability to browse the internet or access new information beyond what I already know. I can try to provide answers to your questions based on my existing knowledge, but if you have a question that is outside of my knowledge base or if you need up-to-date information, I may not be able to help you. Is there anything else I can assist you with?

**Comment by face-hh** [comment]
I appreciate the pull request, @Iliannnn 🙏.

@Assassin-1234 please refrain from bringing political into the conversation, alongside off-topic comments, thanks!

**Pull request merged**